marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/LoadUserArtifactsCommand.java

```diff
@@ -137,7 +137,7 @@ public String toURI(Resource r) throws IOException {
         };
         ResourceToURI stepResourceToURI = new ResourceToURI(){
             public String toURI(Resource r) throws IOException {
-                return "/steps/" + r.getFile().getParentFile().getParentFile().getName() + "/" + r.getFile().getParentFile().getName() + "/" + r.getFilename();
+                return "/step-definitions/" + r.getFile().getParentFile().getParentFile().getName() + "/" + r.getFile().getParentFile().getName() + "/" + r.getFilename();
```

Comment:
Should the collection name change to http://marklogic.com/data-hub/step-definition ? If so, there are quite a few places in enode code that has to be changed as well.

Reply:
I've added 
`// TODO: Rename collection URI after changing server code`
for that change as I dont want to break the enode code.
Should be taken up as part of https://project.marklogic.com/jira/browse/DHFPROD-2096

Reply:
But is the change from step to step-definition part of the scope of https://project.marklogic.com/jira/browse/DHFPROD-2096 ? I thought DHFPROD-2076 handles that .

Reply:
Can be taken up as part of DHFPROD-2076 as well, but I don't want to break the code for now.